### ISSUE TITLE
runtime of a failed job is the time since the job started

### ISSUE THREAD

**Issue by carandraug** [codalab/codalab-competitions#3232]
In the admin monitoring links, there is a a job status page, accessed via `health/simple_status` URL. This shows all the jobs, their status, and the runtime.  In the case of failed jobs the displayed time is the time since the job started.  For example, if the failed job started 10 hours ago, the runtime shows 10 hours. Here's a screenshoot

![failed-jobs](https://user-images.githubusercontent.com/916140/190525158-998429f2-cc7e-475f-85ec-d6207dad1528.png)

I guess it should either show the time it took for the job to fail or it should show invalid.

**Comment by Didayolo** [comment]
Good catch! I think it should show the time elapsed before the failure.
